streaming/package.json

```diff
@@ -12,7 +12,8 @@
     "url": "https://github.com/mastodon/mastodon.git"
   },
   "scripts": {
-    "start": "node ./index.js"
+    "start": "node ./index.js",
+    "check:types": "tsc --noEmit"
```

Comment:
@renchap I am dubious of the value this brings right now, so let's not make it gating.

Reply:
I never asked to have it ran anywhere, my goal was to have a way for devs to run it and try to get the number of errors down :)